libs/polkadot-api-provider/src/webb-provider.ts

```diff
@@ -156,6 +160,11 @@ export class WebbPolkadot
     return this.typedChainId;
   }
 
+  async getResourceId(): Promise<ResourceId | null> {
+    throw new Error('Method not implemented.');
+    return null;
+  }
```

Comment:
Can this be removed if its not implemented?

Reply:
We have to have this method because this method is from the [`WebbApiProvider`](https://github.com/webb-tools/webb-dapp/blob/7950409df0004143e06fd69cfdf45d372c64af86/libs/abstract-api-provider/src/webb-provider.interface.ts#L202) interface that is shared between Web3 API and Polkadot API. In the child class we have to implement all methods from the interface.